### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by Gunjuzone**
In this Pull Request, more features were added to my Portfolio

1. Added Recent Work section
2. Added Professional Competency Section
3. Added About Me Section.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -24,14 +24,15 @@ header {
   font-style: normal;
```
Comment (mricanho) [inline style.css:25]: - [ ] In the body, the browser gives an automatically 8px margin, this creates an error in the design, you should add a CSS reset file to fix this. Check more [info here](https://stackoverflow.com/questions/12582624/what-is-a-user-agent-stylesheet)

WhatsApp Image 2021-06-10 at 18 57 21

image

**Here is your [live demo](https://rawcdn.githack.com/Gunjuzone/My-Portfolio/4c4af3ee599686eb941e7a8791a0eaccd99b474d/index.html)**

File: index.html [hunk 2]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
 		 
 		</div>
 	</section>
-    
+  <section>
+		<div class="works">
+			<h2 class="recent">My Recent Works</h2>
+			<img src="Assets/backgroundcard.png" alt="">
```
Comment (mricanho) [inline index.html]: - [ ] The section must have the proper background image (decorations are always background images, not `img` tags), if an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info

File: index.html [hunk 3]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
+				no accounts or sign-ups required. has been 
+				the industry's standard</p>
+
+				<ul class="btn">
+					<li class="btn-skills">html</li>
+					<li class="btn-skills">bootstrap</li>
+					<li class="btn-skills">Ruby</li>
+				</ul>
+	
+			<ul class="btn-mid">
```
Comment (mricanho) [inline index.html]: - [ ] Avoid this tag (ul) for a list that is not a list. You can tag this as a div.

File: index.html [hunk 4]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
+				<li class="btn-see-project"><a href="#">See Project</a></li>
```
Comment (mricanho) [inline index.html]: - [ ] Use the proper `button` tag

File: index.html [hunk 5]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
+				<li class="btn-skills">Ruby</li>
+			</ul>
+
+		<ul class="btn-mid">
+			<li class="btn-see-project"><a href="#">See Project</a></li>
+		</ul>
+	</div>
+	</section>
+
+	<section>
```
Comment (mricanho) [inline index.html]: - [ ] All these section tags must be gone. The list of projects uses a ul tag to group them, and li per project. Please change it.

File: index.html [hunk 6]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
+					<li class="btn-skills">Ruby</li>
+				</ul>
+	
+			<ul class="btn-mid">
+				<li class="btn-see-project"><a href="#">See Project</a></li>
+			</ul>
+		</div>
+	</section>
+
+	<section>
```
Comment (mricanho) [inline index.html]: - [ ] All these section tags must be gone. The list of projects uses a ul tag to group them, and li per project. Please change it.

File: index.html [hunk 7]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
+					<li class="btn-skills">bootstrap</li>
+					<li class="btn-skills">Ruby</li>
+				</ul>
+	
+			<ul class="btn-mid">
+				<li class="btn-see-project"><a href="#">See Project</a></li>
+			</ul>
+		</div>
+	</section>
+	<section>
```
Comment (mricanho) [inline index.html]: - [ ] All these section tags must be gone. The list of projects uses a ul tag to group them, and li per project. Please change it.

File: index.html [hunk 8]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
```
Comment (mricanho) [inline index.html]: - [ ] All these section tags must be gone. The list of projects uses a ul tag to group them, and li per project. Please change it.

File: index.html [hunk 9]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
+			the industry's standard</p>
+
+			<ul class="btn">
+				<li class="btn-skills">html</li>
+				<li class="btn-skills">bootstrap</li>
+				<li class="btn-skills">Ruby</li>
+			</ul>
+
+		<ul class="btn-mid">
+			<li class="btn-see-project"><a href="#">See Project</a></li>
```
Comment (mricanho) [inline index.html]: - [ ] Use the proper button tag

File: index.html [hunk 10]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
+				the industry's standard</p>
+
+				<ul class="btn">
+					<li class="btn-skills">html</li>
+					<li class="btn-skills">bootstrap</li>
+					<li class="btn-skills">Ruby</li>
+				</ul>
+	
+			<ul class="btn-mid">
+				<li class="btn-see-project"><a href="#">See Project</a></li>
```
Comment (mricanho) [inline index.html]: - [ ] Use the proper button tag

File: index.html [hunk 11]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
```
Comment (mricanho) [inline index.html]: - [ ] Use the proper button tag

File: index.html [hunk 12]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
```
Comment (mricanho) [inline index.html]: - [ ] Use the proper button tag

File: index.html [hunk 13]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
```
Comment (mricanho) [inline index.html]: - [ ] Use the proper button tag

File: index.html [hunk 14]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
+  <div class="about-me">
+    <h2>About me</h2>
+      
+    <p>Hello I’m a software developer! I can help you build a product,feature 
+      or website look through some of my work and experience!If you like what you see 
+      and have a project you need coded, don’t hestiate to contact me.
+    </p>
+      
+    <ul class="btn-mid-about">
+		<li class="btn-see-about"><a href="#">Get My Resume</a></li>
```
Comment (mricanho) [inline index.html]: - [ ] Use the proper button tag

File: index.html [hunk 15]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
+			no accounts or sign-ups required. has been 
+			the industry's standard</p>
+
+			<ul class="btn">
+				<li class="btn-skills">html</li>
+				<li class="btn-skills">bootstrap</li>
+				<li class="btn-skills">Ruby</li>
+			</ul>
+
+		<ul class="btn-mid">
```
Comment (mricanho) [inline index.html]: - [ ]  Avoid this tag (ul) for a list that is not a list. You can tag this as a div.

File: index.html [hunk 16]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
+				no accounts or sign-ups required. has been 
+				the industry's standard</p>
+
+				<ul class="btn">
+					<li class="btn-skills">html</li>
+					<li class="btn-skills">bootstrap</li>
+					<li class="btn-skills">Ruby</li>
+				</ul>
+	
+			<ul class="btn-mid">
```
Comment (mricanho) [inline index.html]: - [ ]  Avoid this tag (ul) for a list that is not a list. You can tag this as a div.

File: index.html [hunk 17]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
```
Comment (mricanho) [inline index.html]: - [ ]  Avoid this tag (ul) for a list that is not a list. You can tag this as a div.

File: index.html [hunk 18]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
```
Comment (mricanho) [inline index.html]: - [ ]  Avoid this tag (ul) for a list that is not a list. You can tag this as a div.

File: index.html [hunk 19]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
```
Comment (mricanho) [inline index.html]: - [ ]  Avoid this tag (ul) for a list that is not a list. You can tag this as a div.

File: index.html [hunk 20]
```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
...
+<section>
+  <div class="about-me">
+    <h2>About me</h2>
+      
+    <p>Hello I’m a software developer! I can help you build a product,feature 
+      or website look through some of my work and experience!If you like what you see 
+      and have a project you need coded, don’t hestiate to contact me.
+    </p>
+      
+    <ul class="btn-mid-about">
```
Comment (mricanho) [inline index.html]: - [ ]  Avoid this tag (ul) for a list that is not a list. You can tag this as a div.

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @Gunjuzone, 

Good job so far! 🔥🔥 You are following the gitflow and your README file it's awesome. Your commit messages are consistent.

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

you can do it

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Gunjuzone) [reply style.css:25]: Hello, would something like this fix the error


* {  margin: 0;  padding: 0;  box-sizing: border-box; }
I dont really understand what the problem is

Comment (mricanho) [reply style.css:25]: > Hello, would something like this fix the error
> 
> * {  margin: 0;  padding: 0;  box-sizing: border-box; }
>   I dont really understand what the problem is

Open[ this link](https://rawcdn.githack.com/Gunjuzone/My-Portfolio/4c4af3ee599686eb941e7a8791a0eaccd99b474d/index.html) in a mobile phone, and you will see a margin in the body that doesn't need to be there. [Learn more](https://stackoverflow.com/questions/11578819/css-reset-what-exactly-does-it-do) about inconsistencies in browsers and how a CSS reset file can help you.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Gunjuzone, 

I'm gonna approve your project because you met all the hardcore requirements for this milestone, although I strongly suggest you add the CSS reset file, adding a margin: 0 to the HTML is not enough to fix the problem with your layout.

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**